### PR TITLE
Update kubectl version in the prow-tests image to v1.18.0

### PR DESCRIPTION
**Pull request by chizhg**
As chatted in https://knative.slack.com/archives/CCSNR4FCH/p1617659140069200?thread_ts=1617650459.068300&cid=CCSNR4FCH

/cc @dprotaso

### CODE REVIEW

File: images/prow-tests/Dockerfile [hunk 1]
```diff
@@ -16,13 +16,19 @@ FROM gcr.io/k8s-testimages/kubekins-e2e:v20210113-cc576af-master AS stable-stuff
 
 ENV DEBIAN_FRONTEND noninteractive
 ARG GCLOUD_VERSION=317.0.0
+ARG KUBECTL_VERSION=v1.18.0
```
Comment (dprotaso) [inline images/prow-tests/Dockerfile:19]: v1.18.7

Probably worth picking up any patches

Comment (chizhg) [reply images/prow-tests/Dockerfile]: Done.

**Comment by dprotaso** [comment]
I guess this needs another approver

**Pull request merged**